src/Common/src/Interop/Windows/mincore/Interop.OutputDebugString.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Interop.Libraries.Debug, CharSet = CharSet.Unicode, EntryPoint="OutputDebugStringW", ExactSpelling=true)]
+        internal static extern void OutputDebugString(string message);
+    }
+}
```

Comment:
I will fix this up for all the cs files.  However, I'll leave the csproj and packages.config files as is, as I don't want to start fighting VS on how it thinks these files should be set up.